.jenkins/infrastructure/docker/build_windows_docker_images.Jenkinsfile

```diff
@@ -46,6 +46,18 @@ pipeline {
                 }
             }
         }
+        stage("Build Windows Server 2022 Docker Image") {
```

Comment:
Running this pipeline revealed an issue with this stage (error below). I'm guessing that we can't run Windows Server 2022 containers on Server 2019. Perhaps we can change the default for AGENT_LABEL to 2022-nonsgx -- assuming Server 2022 is able to run Server 2019 containers.
```
Step 1/4 : FROM mcr.microsoft.com/windows/servercore:ltsc2022
ltsc2022: Pulling from windows/servercore
no matching manifest for windows/amd64 10.0.17763 in the manifest list entries
```

Reply:
Okay, lets test it that way